components/brave_sync/brave_profile_sync_service_impl.cc

```diff
@@ -415,20 +428,20 @@ void BraveProfileSyncServiceImpl::OnGetInitData(
 
   client_data::Config config;
   config.api_version = brave_sync_prefs_->GetApiVersion();
-  config.server_url = "https://sync.brave.com";
+  config.server_url = "https://sync-staging.brave.com";
```

Comment:
should this point to staging?

Reply:
I will change it along with brave/sync bump once the PR in that repo is merged

Reply:
we should convert this to a gn config